s1tbx-op-insar-ui/pom.xml

```diff
@@ -91,6 +101,11 @@
             <groupId>org.esa.snap</groupId>
             <artifactId>snap-product-library-ui</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.esa.snap</groupId>
+            <artifactId>snap-product-library-ui-v2</artifactId>
+            <version>${project.version}</version>
+        </dependency>
         <dependency>
```

Comment:
duplicate dependency

Reply:
I opened the pom.xml file and check how many times is defined the dependency searching by 'snap-product-library-ui-v2'.
Only one occurrence exists in the file.